qemu/tests/enable_scatter_windows.py

```diff
@@ -172,17 +175,32 @@ def _get_driver_version(session):
         lambda: _is_process_finished(session, autoit_name), timeout, 20, 3
     )
 
+    error_context.context("Check if WinDump is installed", test.log.info)
+    check_result = session.cmd_output(check_windump_installed_cmd)
+    if windump_name not in check_result:
+        error_context.context("Install WinDump", test.log.info)
+        windump_install_cmd = utils_misc.set_winutils_letter(
+            session, windump_install_cmd
+        )
+        status, output = session.cmd_status_output(windump_install_cmd, timeout=timeout)
+        if status != 0:
+            test.error(
+                "Failed to install WinDump, status=%s, output=%s" % (status, output)
+            )
+    else:
+        test.log.info("WinDump is already installed")
+
     error_context.context("Check if wireshark is installed", test.log.info)
-    check_installed_cmd = params.get("check_installed_cmd")
-    check_result = session.cmd_output(check_installed_cmd)
+    check_wireshark_installed_cmd = params.get("check_wireshark_installed_cmd")
+    check_result = session.cmd_output(check_wireshark_installed_cmd)
     if "tshark" not in check_result:
         error_context.context("Install wireshark", test.log.info)
-        install_wireshark_cmd = params.get("install_wireshark_cmd")
-        install_wireshark_cmd = utils_misc.set_winutils_letter(
-            session, install_wireshark_cmd
+        wireshark_installation_cmd = params.get("wireshark_installation_cmd")
+        wireshark_installation_cmd = utils_misc.set_winutils_letter(
+            session, wireshark_installation_cmd
         )
         status, output = session.cmd_status_output(
-            install_wireshark_cmd, timeout=timeout
+            wireshark_installation_cmd, timeout=timeout
         )
         if status:
             test.error(
```

Comment:
Could you try to implement them into a for loop? e.g. for tool_name in ("windump", "wireshark") or it could be a dict as you might want to name a different prefix instead of the tool name itself.